### ISSUE TITLE
[Accessibility] The color contrast ratio of the focus indicator is not enough for icons in navigation bar

### ISSUE THREAD

**Issue by jdneo** [eclipse-platform/eclipse.platform.ui#4026]
Let's make sure issue is not already fixed in latest builds first.

- [x] I verified I can reproduce this issue against [latest Integration Build of Eclipse SDK](https://download.eclipse.org/eclipse/downloads/)

## Steps to reproduce

From a fresh installation and clean workspace:

1. Hover on an icon on a view's navigation bar
2. Observe the color of the hover background

<img width="1301" height="880" alt="Image" src="https://github.com/user-attachments/assets/3c57d872-7cd2-4881-9794-5684d906dbde" />

## Tested under this environment:
Windows 11
Version: 2026-03 (4.39.0)
Build id: 20260305-0817

https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

## Community

- [x] I understand reporting an issue to this OSS project does not mandate anyone to fix it. Other contributors may consider the issue, or not, at their own convenience. The most efficient way to get it fixed is that I fix it myself and contribute it back as a good quality patch to the project.

**Comment by merks** [comment]
I find the contrast on the hovers on the native File Explorer even less distinct than the hovers on the IDE toolsbars

<img width="317" height="144" alt="Image" src="https://github.com/user-attachments/assets/ea7f615a-fac1-488d-ac6f-0ca76144de68" />

Can you share the same assessment of that as a point of reference?  Maybe it's different because the background it white and not gray?

**Comment by kupatkar99** [comment]
In the screenshot above, when the focus is on the copy button, the "copy" tooltip appears, so the user can identify the focus location. Therefore, this would not be considered a high severity bug.

**Comment by merks** [comment]
The IDE tool buttons also display hover text:

<img width="266" height="76" alt="Image" src="https://github.com/user-attachments/assets/c39e4c56-204f-45ad-bb55-af2e8703a9cd" />

And of course the mouse is visible as well, it just doesn't screen capture...

This also seems extremely subtle with no hover text either:

<img width="141" height="169" alt="Image" src="https://github.com/user-attachments/assets/62564939-4fe0-4032-8593-f3d97e1849ea" />

So I remain curious how these measure up in a comparable assessment.

**Comment by HeikoKlare** [comment]
To the best of my knowledge, this is specific to Windows and these colors come from Windows itself. For reference, you can take a look at other applications using the win32 controls (such as Notepad++). Note that, for example, on MacOS there is no feedback on hover at all. And the selection feedback is inferior to the current one on Windows as well (just recoloring the icon instead of coloring the background).

I guess this could only be solved (for Windows) by doing custom painting of the highlight also for the default theme, similar to what to my understanding is done for the dark theme:
- https://github.com/eclipse-platform/eclipse.platform.swt/pull/3202

**Comment by merks** [comment]
Here too with the tiny icons in notepad on Windows 11 it's really hard to see any hover feedback and in fact the little circled person does not provide any hover feedback:

<img width="214" height="52" alt="Image" src="https://github.com/user-attachments/assets/e754d320-0e27-4338-8bd1-cb9d5f3220ba" />

Unless someone presents some compelling reason why it's a problem for the Eclipse IDE but not all these other places, I wold conclude that it's just not a compelling problem.

**Comment by jdneo** [comment]
@kupatkar99 Please check the comments and make a decision whether this issue is still valid. Thanks.

**Comment by merks** [comment]
Given the lack of additional comments, I'll close this.
